spring-apps/autosleep-core/src/main/java/org/cloudfoundry/autosleep/ui/web/model/OrgEnrollmentConfigRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.cloudfoundry.autosleep.ui.web.model;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+import lombok.AccessLevel;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder
+@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@NoArgsConstructor(access = AccessLevel.PRIVATE)
+public class OrgEnrollmentConfigRequest {
+
+    @JsonProperty("idle-duration")
+    private String idleDuration;
+
+    @JsonProperty("exclude-spaces-from-auto-enrollment")
+    private String excludeSpacesFromAutoEnrollment;
+
+    @JsonProperty("state")
+    private String state;
```

Comment:
@gberche-orange The enum variable names are ok. I have created a story https://www.pivotaltracker.com/story/show/137034845 to track and close the discussion.